ironfish/src/wallet/account.ts

```diff
@@ -204,6 +204,40 @@ export class Account {
     return assetBalanceDeltas
   }
 
+  async saveAssetFromChain(
+    {
+      createdTransactionHash,
+      id,
+      metadata,
+      name,
+      owner,
+    }: {
+      createdTransactionHash: Buffer
+      metadata: Buffer
+      id: Buffer
+      name: Buffer
+      owner: Buffer
+    },
```

Comment:
This looks a little odd; can we pass each field in as a typed argument? Or use the type returned by `chain.getAssetById`?